src/ActionsImporter/Commands/Bitbucket/Common.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.CommandLine;
+
+namespace ActionsImporter.Commands.Bitbucket;
+
+public static class Common
+{
+    public static readonly Option<string> InstanceUrl = new("--bitbucket-instance-url")
+    {
+        Description = "The URL of the Bitbucket instance.",
+        IsRequired = false,
+    };
```

Comment:
```suggestion
```

Do we need to have the instance-url since it will always be `https://bitbucket.org/`?  If you do remove this I think you need to remove the references in all of the commands